app/grants/templates/grants/cart/eth.html

```diff
@@ -173,8 +171,7 @@ <h4 class="col gc-font-base text-dark p-0">Summary</h4>
   <!-- Checkout buttons -->
   <div class="row justify-content-end">
     <div class="btn-group flex-column flex-grow-1 flex-md-grow-0 flex-md-row" role="group" aria-label="Checkout buttons">
-      
-      {% if is_staff %}
```

Comment:
^ I'd say let's keep this on until we get the green sign from the team that this is good for rollout